### PR TITLE
I57 new summary table

### LINKED ISSUE #57: New summary table for status overview
In run-regional-updates handle a new csv (status.csv?) that contains the latest summary for each location when processing the run outcome. This may also help with #54 .

Perhaps the following columns:
| Dataset | subregion | last run timestamp | last run status | latest results generated at | latest results data up to |
| -------- | ----------- | --------------------- | --------------- | ----------------------------- | -------------------------- |
| united-kingdom | * | 2020-09-30 13:45:53 | No New Data Available | 2020-09-29 05:34:23 | 2020-09-25 |
| united-kingdom | London | 2020-09-30 13:45:53 | No New Data Available | 2020-09-29 05:34:23 | 2020-09-25 |

last run status = Success | Error | Timed Out | No New Data Available (I think we should be able to detect all these conditions)

### PR DESCRIPTION
**Pull request by JAllen42**
closes #57

### CODE REVIEW

**Review by seabbs — APPROVED** [review]
This all looks pretty reasonable to me! Thoughts @joeHickson?

File: R/run-region-updates.R [hunk 1]
```diff
@@ -179,19 +153,153 @@ rru_log_outcome <- function(outcome) {
...
+      status_row$latest_results_successful_regions <- ifelse(dataset_processed, dataset_counts$successes, status_row$latest_results_successful_regions)
+      status_row$latest_results_timing_out_regions <- ifelse(dataset_processed, dataset_counts$timeouts, status_row$latest_results_timing_out_regions)
+      status_row$latest_results_failing_regions <- ifelse(dataset_processed, dataset_counts$failures, status_row$latest_results_failing_regions)
+      status_row$oldest_region_results <- outcome[[dataset_name]]$oldest_results
+      status <- dplyr::rows_upsert(status, status_row, by = c("dataset"))
+    }
   }
   futile.logger::flog.trace("writing file")
-  write.csv(stats, file = filename, row.names = FALSE)
+  write.csv(stats, file = stats_filename, row.names = FALSE)
```
Comment (joeHickson) [inline R/run-region-updates.R:222]: @JAllen42 how hard would it be to sort these before saving them? Region / subregion alphabetically would be awesome ;)

**Comment by joeHickson** [comment]
I wrote half of it so it looks awesome ;)
Just kidding - I think it should be fine. I have asked a wishlisty type comment of Jamie (or you may be able to slip it in) about sorting the data before saving.

Overwrite runtimes.csv with the latest - that's a test run.

**Comment by joeHickson** [comment]
go for it

**Comment by seabbs** [comment]
Do you have a local copy of the repo? Can you resolve the conflict locally?

**Comment by joeHickson** [comment]
I can - I need to scoot for nursery pickup but I'll get to it tomorrow if I can

**Comment by seabbs** [comment]
awesome thanks! If I get a chance will have a crack but can't bear to tax the internet connection with the clone....

**Comment by JAllen42** [comment]
I've tried to merge master in, but I think I might have messed it up. Some files, like entities.R don't look like how I was expecting, so it might be helpful if someone could review the last couple of commits where I tried to do the merge

**Comment by joeHickson** [comment]
I'm taking a look now @JAllen42

**Comment by joeHickson** [comment]
Thanks for flagging that @JAllen42  - that would have broken immediately upon running. All fixed now.

**Comment by joeHickson** [comment]
snuck the sorting in ;)

**Comment by seabbs** [comment]
Good to merge?

**Pull request merged**